packages/components/table/src/util.ts

```diff
@@ -269,13 +269,7 @@ export function toggleRowStatus<T>(
     } else if (!newVal && included) {
       toggleStatus('remove')
     }
-  } else {
-    if (included) {
-      toggleStatus('remove')
-    } else {
-      toggleStatus('add')
-    }
-  }
+  } else included ? toggleStatus('remove') : toggleStatus('add')
```

Comment:
Do you think this is simpler and easier to understand than before?

Reply:
yes, more understandable

Reply:
I searched the project and found only three cases where brackets were omitted when using `if else` (there are only simple calls internally). It is weird for you to use it like this.

Reply:
maybe it should included in eslint